tools/install-powershell.ps1

```diff
@@ -0,0 +1,114 @@
+<#
+.Synopsis
+    Install PowerShell Core on Windows.
+.DESCRIPTION
+    By default, the latest PowerShell Core release package will be installed.
+    If '-Daily' is specified, then the latest PowerShell Core daily package will be installed.
+.Parameter Destination
+    The destination path to install PowerShell Core to.
+.Parameter Daily
+    Install PowerShell Core from the daily build.
+    Note that the 'PackageManagement' module is required to install a daily package.
+.Parameter NotOverwrite
+    Do not overwrite the destination folder if it already exists.
+.Parameter AddToPath
+    Add the absolute destination path to the 'User' scope environment variable 'Path'
+#>
+[CmdletBinding()]
+param(
+    [Parameter()]
+    [ValidateNotNullOrEmpty()]
+    [string] $Destination,
+
+    [Parameter()]
+    [switch] $Daily,
+
+    [Parameter()]
+    [switch] $NotOverwrite,
+
+    [Parameter()]
+    [switch] $AddToPath
+)
+
+Set-StrictMode -Version latest
+$ErrorActionPreference = "Stop"
+
+if (-not $Destination) {
+    $Destination = "~/pscore"
```

Comment:
Should we default to `~/.pscore` following `.dotnet` convention?

Reply:
I don't have a preference here. But on windows `dotnet-install.ps1` actually installs dotnet in `$env:LOCALAPPDATA\Microsoft\dotnet\`, only the some helping files are placed in `~/.dotnet`.

Reply:
Perhaps we should use `$env:LOCALAPPDATA\Microsoft\powershell`

Reply:
OK, will change to that path.